controller.js

```diff
@@ -0,0 +1,33 @@
+const input =$('#input')
+const adPost= function(){
+    tweeter.addPost(input.val())
+    input.val('')
+
+    Renderer()
+}
+$('#posts').on('click','.commentAdd', function() {
+    let pstId=$(this).closest(".post").attr('id')
```

Comment:
It is better to name variables what they actually are. Even though `pstId` is short, short is not always better. You want people to be able to look at your code and instantly understand what you are writing. So instead name variables the full name `postId`